app/controllers/master/events/imports_controller.rb

```diff
@@ -13,6 +13,7 @@ def create
   private
 
   def check_shared_secret
-    head(403) unless params[:secret] == Events.shared_secret
+    permitted_params = params.permit(%i[secret]).to_h
```

Comment:
Why `to_h`?

Reply:
@hallelujah we (@jsmadis  and @josemigallas )discussed in the call that its safe to convert to hash , so if we permit we convert it to hash.

Reply:
@hallelujah PR resolved